config/collector.yaml

```diff
@@ -33,11 +68,23 @@ processors:
         - otelcol_process_memory_rss
         - otelcol_grpc_io_client_completed_rpcs
         - otelcol_googlecloudmonitoring_point_count
+
+  # The batch processor is in place to regulate both the number of requests
+  # being made and the size of those requests.
```

Comment:
Cloud Monitoring recommends using a batch size of 200, which is the maximum batch size allowed.

Reply:
Addressed in [b4875a4](https://github.com/GoogleCloudPlatform/otlp-k8s-ingest/pull/52/commits/b4875a4ae0cefa117e6dcd2a5b9309a31395c939)